deploy.sh

```diff
@@ -44,7 +44,11 @@ cd $DEST_DIR
 # Transform the readme
 if [ -f readme.md ]; then
 	mv readme.md readme.txt
-	sed -i '' -e 's/^# \(.*\)$/=== \1 ===/' -e 's/ #* ===$/ ===/' -e 's/^## \(.*\)$/== \1 ==/' -e 's/ #* ==$/ ==/' -e 's/^### \(.*\)$/= \1 =/' -e 's/ #* =$/ =/' readme.txt
+	if [ -f CHANGELOG.md ]; then
```

Comment:
Maybe something like:

```
 if [ -f readme.md ]; then
  readmeMd='readme.md'
elif [ -f README.md ]; then
  readmeMd='readme.md'
else
  echo "Cannot find readme markdown file";
  exit;
fi
```

And then do something similar for the changelog.md file...then update the related code to match that idea?